### DISCUSSION TITLE
Docusaurus 自动化生成文章总结 | Alan Wang

### DISCUSSION THREAD

**Discussion by giscus** [3Alan/site#51] [Announcements]
# Docusaurus 自动化生成文章总结 | Alan Wang

利用 Gemini AI 搭配 Github Bot 实现自动化生成文章总结

https://www.alanwang.site/posts/blog-guides/docusaurus-ai-bot

**Comment by quan7u** [comment]
👍酷

**Comment by tysaner** [comment]
大佬，我按照你的步骤一步一步来的，最后又两个问题，能不能要个联系方式，帮忙解决一下呀，栓Q
